deepl-connector-product/zip.xml

```diff
@@ -20,6 +20,7 @@
       <outputDirectory>/</outputDirectory>
       <includes>
         <include>README.md</include>
+                <include>README_DE.md</include>
```

Comment:
please use the same intend as the existing pom.xml, to keep the formatting in harmony